LICENSE.md

```diff
@@ -58,8 +58,8 @@ The User accepts the following liability:
 3. Right to use
 --------
 
-The User is allowed to install the Mod and play without restriction.  
-The User may not limit access to the Mod in any way, except as provided by the Game.  
+The User is allowed to install the Mod and play it provided the following conditions are met:
+ - If the User provides the Mod for others to play (for example, on a multiplayer Game server) the user may not limit other users' access to the Mod in any way, except as provided by the Game.  
```

Comment:
Again, I made the change because of a contradiction. You can't on the one hand grant rights to play without restriction, and then on the other hand later say that the player can't hide features. Maybe this isn't the best way to resolve that contradiction, but it's one attempt.